resources/js/data-access/memo-api/groups/Admin.ts

```diff
@@ -43,14 +49,14 @@ export namespace Admin {
   export const usersQueryOptions = (ids?: ListInParam) => {
     const request = createListRequest(ids);
     return {
-      queryFn: () => getUsersList(request),
+      queryFn: ({signal}) => getUsersList(request, {signal}),
```

Comment:
ten kod chyba nieużywany nigdzie?

a jak w ogóle ten signal tu trafia i skąd?

Reply:
Ten usersQueryOptions się przekazuje do createQuery i to on woła to queryFn, i przekazuje tam różne rzeczy. Ja tak naprawdę tutaj dodaję ten signal do tych zapytań gdzie Ryszard tego nie dodał. I ja już raz na oczy przy testach widziałem scancelowane zapytanie więc wierzę że jakoś tam to działa :) Możesz poczytać więcej na [TanStack Query](https://tanstack.com/query/v5/docs/react/overview).

A ten kod jest w sumie faktycznie trochę na zapas bo edytowanie użytkownika używa tego userQueryOptions poniżej, które robi to samo zapytanie niby, ale owinięte. No ale dodane dla symetrii.

Reply:
Ok, ale usersQueryOptions możnaby usunąć skoro martwy kod. Jak będzie potrzebne to można dodać symetrycznie.

Reply:
Aha, no teraz to akurat jest użyte do pobrania wszystkich użytkowników w tym users_fake_tquery. Będzie można usunąć jak się usunie fake jak się przepnie użytkowników na prawdziwe tquery.